tests/ops/functions/conftest.py

```diff
@@ -57,7 +57,9 @@
     (qml.s_prod(1.1, qml.RX(1.1, 0)), {}),
     (qml.prod(qml.PauliX(0), qml.PauliY(1), qml.PauliZ(0)), {}),
     (qml.ctrl(qml.RX(1.1, 0), 1), {}),
-    (qml.exp(qml.PauliX(0), 1.1), {}),
+    (qml.exp(qml.PauliX(0), 1.1), {"skip_differentiation": True}),
+    (qml.exp(qml.PauliX(0), 2.9j), {}),
```

Comment:
I'd be ok with commenting this line out till we figure out what to do with the generator of `Exp`.
```suggestion
    #(qml.exp(qml.PauliX(0), 2.9j), {}),
```

Reply:
I can xfail it and create an issue to track it?